include/openssl/evp.h

```diff
@@ -349,6 +351,8 @@ int (*EVP_CIPHER_meth_get_ctrl(const EVP_CIPHER *cipher))(EVP_CIPHER_CTX *,
 /* Set the input buffer lengths to use for a pipelined operation */
 # define         EVP_CTRL_SET_PIPELINE_INPUT_LENS        0x24
 
+# define         EVP_CTRL_GET_IVLEN                      0x25
```

Comment:
If we're going to make changes to a public header in 1.1.1, then the same changes will need to be forward ported to master!

Reply:
...  I hadn't even noticed this was for 1.1.1.  Yeah, I agree with that.

Reply:
So should we fix this in 1.1.1? or are you suggesting making it internal by reserving some ctrl_id and flag?

Reply:
What I hear from @mattcaswell isn't that we should hide this, but that `EVP_CTRL_GET_IVLEN` needs to be handled in master too.  Do you see that as a bit problem?

Reply:
No it will be fine - I thought I would do the 1.1.1 version, to see if you guys thought it is worth doing or not. If it is I will do a master variant afterwards.

Reply:
PR for master is listed below